#!/usr/bin/python3

def raise_exception_msg(message=""):
    """raise exception with message

    Args:
        message: the message for the error message

    Returns:
        None
    """
    raise NameError(message)
